priv/posts/marketing/20241121_getting-to-know-us-kiko-fernandez-reyes.md

```diff
@@ -0,0 +1,93 @@
+
+Another challenge for me is that, as a board member of the Erlang Ecosystem Foundation, I’m trying to bring research back to Erlang. Many communities heavily benefit from research in languages like Haskell and OCaml, but I don’t believe there’s as much research in Erlang. I’m exploring ways for the Erlang/OTP team to collaborate with various research institutions.
+
+We’ve submitted one EU project proposal, as well as some others locally here in Sweden. So far, they’ve been rejected, but that doesn’t mean we won’t get there. It just means we need to work a bit harder on them. I believe it’s only a matter of time before we can attract researchers back to Erlang to foster more innovation within the BEAM.
+
+#### You're co-chairing the Erlang Workshop for the second year in a row. What has this experience been like, and why do you believe it's important to participate? 
+
+This is my second year, and I hope to continue for a third, but that depends on the steering committee. The experience has been great; it involves a lot of work on the organizational side. My co-chair, [Laura Voinea](https://scholar.google.co.uk/citations?hl=en&user=aXxC7OcAAAAJ "Laura Voinea"), and I work very well together, which I believe is reflected in the quality of this year's workshop. We received numerous high-quality submissions, and I’m really happy with how everything has turned out.
+
+I believe there are several reasons why participation is important. If you work in **industry**, it would be beneficial to submit papers that showcase proven patterns and solutions. I don’t think any solution needs to be perfect. You can submit something along the lines of, “For this scenario, we know how to implement a solution that works quite well.” I think that’s a valuable contribution because others can read and reference your paper, allowing them to address challenges you may not have known how to handle.. It's really great to have use cases.
+
+From the **academic** point of view, I think if you want to know if your research is relevant for industry, this is a place to go and check. You can just work all you want on the theory, and that is a really important part. But if you want to see if this is applicable to a concurrent and distributed programming language, the Erlang Workshop is probably where you need to be. You can submit ongoing work and interact with industry and academics. You're going to get lots of feedback. In this year’s Workshop, there were great discussions. 
+
+Finally, as a **participant**, if you want to know more or less what is the latest that's going on in the BEAM ecosystem, then you should participate. You are going to make academic contacts, in case you need the part of the Innovation side. And if not, you're still going to meet plenty of people from industry. I think it's really important to continue building this up so that we can do great things together.

```

Comment:
```suggestion
Finally, as a **participant**, if you want to know more or less what is the latest that's going on in the BEAM ecosystem, then you should participate. You are going to make academic contacts, in case you need that part of the innovation side. And if not, you're still going to meet plenty of people from industry. I think it's really important to continue building this up so that we can do great things together.

```